### DISCUSSION TITLE
TypeError: Cannot read properties of null (reading 'removeChild')     at ue.remove (petite-vue.es.js?module:1:19221)

### DISCUSSION THREAD

**Discussion by sureshvv** [vuejs/petite-vue#236] [General]
I am getting this error. Unable to trace the location in my code.

Any help appreciated.

**Comment by antman3351** [comment]
This may or may not help 😅

I was getting the same error ( from a different line ) from  `this.template.parentNode.removeChild(this.template);`. The issue was changing a source ( array ) of a drop-down `<select>` with a new array and appending an item right after. I'm guessing it causes a double event of building the `<select>` options
